### PR TITLE
Update all depedencies

### PR DESCRIPTION
**Pull request by XVincentX**
Our package are a bit behind. This PR will bring us to the present again.

I've gone through all the changelogs. Things should be fine (TM)

### CODE REVIEW

File: bin/environment.js [hunk 1]
```diff
@@ -140,10 +140,6 @@ exports.bootstrap = (eg, adapter) => {
     .demandCommand()
     .recommendCommands()
     .strict()
-    .alias('h', 'help')
```
Comment (kevinswiber) [inline bin/environment.js]: What's the impact of this block of deletions?

Comment (XVincentX) [reply bin/environment.js]: According to the `yargs` changelog, `--help, -h and --version` are now automatically added. It was a boilerplate for everybody and they decided to put it as default.

Comment (kevinswiber) [reply bin/environment.js]: Oh, I just saw that help and version are included by default.  However, I don't think `-h` is aliased by default, so we may need that.

Comment (XVincentX) [reply bin/environment.js]: That's true. However I also realised that alias is automatically added [here](https://github.com/ExpressGateway/express-gateway/blob/master/bin/eg-generator.js#L60)

image

Comment (kevinswiber) [reply bin/environment.js]: Just tried locally.  I get this:

```
➜ ./bin/index.js -h    
<snip>

Not enough non-option arguments: got 0, need at least 1
```

`--help` works, though.

Comment (XVincentX) [reply bin/environment.js]: The output on my machine is the same. Yargs is complaining of the "non-option" arguments.

image

**Comment by XVincentX** [comment]
@kevinswiber You were right, I made the changes in the wrong part. Thanks for spotting this.

3e45da9 should fix the issue you were pointing.

**Pull request merged**